server/forms.py

```diff
@@ -446,6 +445,8 @@ class MossSubmissionForm(BaseForm):
     file_regex = StringField('Regex for submitted files', default='.*',
                              validators=[validators.required()])
     language = SelectField('Language', choices=[(pl, pl) for pl in COMMON_LANGUAGES])
+    subtract_template = BooleanField('Subtract Template', default=False,
```

Comment:
I think we should leave it off by default (from my testing) - It just seems really good at catching last year's template. I have to evaluate it some more.